nntrainer/models/neuralnet.cpp

```diff
@@ -849,27 +849,33 @@ std::vector<float *> NeuralNetwork::incremental_inference(
   unsigned int step = from ? 0 : to - 1;
 
   for (auto &out : output_tensors) {
-    const auto &out_t = *out.get();
-    float *last_out_buf_data = new float[batch_size * out_t.width()];
+    auto out_t = *out.get();
```

Comment:
is there any reason why you changed it to `auto`? 
If out_t is not modified, `const auto &` would be a better option to avoid the coverity issue.

Reply:
Unfortunately there might be a chance that out_t could be change.
Please check line 876.